pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+asyncio_mode = auto
+asyncio_default_fixture_loop_scope = "function"
```

Comment:
and without this I get:
![Screenshot 2025-02-27 at 1 18 34 pm](https://github.com/user-attachments/assets/59af5102-a2c5-4989-b1f5-1013dd81a136)